agent/engine/docker_task_engine_test.go

```diff
@@ -275,7 +277,7 @@ func TestTaskWithSteadyStateResourcesProvisioned(t *testing.T) {
 	sleepTask := testdata.LoadTask("sleep5")
 	sleepContainer := sleepTask.Containers[0]
 	sleepContainer.TransitionDependenciesMap = make(map[apicontainer.ContainerStatus]apicontainer.TransitionDependencySet)
-	sleepContainer.BuildContainerDependency("pause", apicontainer.ContainerRunning, apicontainer.ContainerPulled)
+	sleepContainer.BuildContainerDependency("pause", apicontainer.ContainerResourcesProvisioned, apicontainer.ContainerPulled)
```

Comment:
Should the desire status set in line 288 also set to ContainerReousrcesProvisioned?